From 25aca632008d7949188975bf72938364205a9fe4 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Thu, 18 Apr 2024 10:54:57 +0200
Subject: [PATCH 591/954] refactor(stm32mp2): rename clock driver

Align with STM32MP13 clock driver naming and use clk-stm32mp2.c.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git dc5b80113c098e45349fe9bdae738d9992cf32a8 refactor(stm32mp2): rename clock driver]
Change-Id: I0ed0c1ce8336a77eb00a3fd1d2efefb22dff420a
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/374676
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
Reviewed-by: Gabriel FERNANDEZ <gabriel.fernandez@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 drivers/st/clk/{stm32mp2_clk.c => clk-stm32mp2.c} | 0
 plat/st/stm32mp2/platform.mk                      | 2 +-
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename drivers/st/clk/{stm32mp2_clk.c => clk-stm32mp2.c} (100%)

diff --git a/drivers/st/clk/stm32mp2_clk.c b/drivers/st/clk/clk-stm32mp2.c
similarity index 100%
rename from drivers/st/clk/stm32mp2_clk.c
rename to drivers/st/clk/clk-stm32mp2.c
diff --git a/plat/st/stm32mp2/platform.mk b/plat/st/stm32mp2/platform.mk
index 70d2f29e46..a7d3920e09 100644
--- a/plat/st/stm32mp2/platform.mk
+++ b/plat/st/stm32mp2/platform.mk
@@ -265,7 +265,7 @@ PLAT_BL_COMMON_SOURCES		+=	drivers/st/iwdg/stm32_iwdg.c				\
 					plat/st/stm32mp2/stm32mp2_syscfg.c
 
 PLAT_BL_COMMON_SOURCES		+=	drivers/st/clk/clk-stm32-core.c				\
-					drivers/st/clk/stm32mp2_clk.c				\
+					drivers/st/clk/clk-stm32mp2.c				\
 					drivers/st/crypto/stm32_rng.c				\
 					drivers/st/crypto/stm32_saes.c
 
-- 
2.53.0

